public/services/utils.ts

```diff
@@ -111,3 +111,11 @@ export const truncate = (input: string, maxLength: number): string => {
   }
   return input
 }
+
+export type Optional<T> = {
```

Comment:
insn't this the same as `Partial<T>` which comes by default with typescript?

Reply:
Looking at the `Partial<T>` type definition, it seems it is in fact exactly the same. Happy to know I wasn't the only one to do this, I suppose? 😆 (I really didn't know so much about TypeScript, and sometime I find easier to do things by myself, but when it already exist, it's good to use it!)